### PR TITLE
Update review-of-international-studies.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via emails. Add quotes for journal title.

### CODE REVIEW

**Comment by POBrien333** [comment]
Another email:
"I just discovered another error in the Review of International Studies style. The first time I cite a particular reference the specific page numbers are not included (only the entire length of articles/book chapters).
For example, I pick this particular book in Zotero and indicate pages 3-4. However, the page numbers do not appear. They only appear the second time I cite this book.
Hedley Bull, The Control of the Arms Race: Disarmament and Arms Control in the Missile Age (New York: Praeger, 1965).
Bull (1965), pp. 3-4.

Does that make sense?"

Hope I fixed that correctly.

**Comment by POBrien333** [comment]
Hi @rmzelle and @adam3smith 
I need your help with this one. when a citation is subsequent with a page locator for only one page it should show as p. 3, but it in fact shows as pp. 3. How do I get this working?

See examples 4-6 below:
1 Hedley Bull, The Control of the Arms Race: Disarmament and Arms Control in the Missile Age (New York: Praeger, 1965), p. 6.
2 Jan Ruzicka and Nicholas J. Wheeler, ‘The puzzle of trusting relationships in the Nuclear Non-Proliferation Treaty’, International Affairs, 86:1 (2010), pp. 69–85; William Walker and Nicholas J. Wheeler, ‘The Problem of Weak Nuclear States’, The Nonproliferation Review, 20:3 (2013), pp. 411–31.
3 Laust Schouenborg, ‘The English School and Institutions: British Institutionalists?’, in Cornelia Navari and Daniel M. Green (eds), Guide to the English School in International Studies (Chichester: Wiley-Blackwell, 2014), pp. 77–90.
4 Walker and Wheeler (2013), pp. 5.
5 Bull (1965), pp. 8.
6 Schouenborg (2014), pp. 12.

File: review-of-international-studies.csl [hunk 1]
```diff
@@ -112,7 +120,7 @@
               <date date-parts="year" form="text" variable="issued" prefix="(" suffix=")"/>
             </group>
             <group delimiter=" ">
-              <label plural="always" variable="page" form="short"/>
+              <label plural="always" variable="locator" form="short"/>
```
Comment (adam3smith) [inline review-of-international-studies.csl]: the `plural="always"` here is your problem. That forces pp. Take it out & this will work as intended

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**